### PR TITLE
CEDS-1544 Removed hmrctest

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: test/base/ViewValidator.scala [hunk 1]
```diff
@@ -32,7 +32,7 @@ trait ViewValidator extends MustMatchers with ViewMatchers {
   private def asDocument(page: String): Document = Jsoup.parse(page)
 
   //TODO Remove methods based on the CSS, stay and use those based on the ID
-  @deprecated("Please use 'page.getElementsBySelector must ...'")
+  @deprecated("Please use 'page.getElementsBySelector must ...'", since = "2019-08-07")
```
Comment (PatrykRudnicki) [inline test/base/ViewValidator.scala]: Using `deprecated` with only one argument is deprecated :D

File: test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala [hunk 2]
```diff
@@ -46,6 +47,9 @@ class CustomsDeclareExportsConnectorIntegrationSpec
   private val notification = Notification("conv-id", "mrn", LocalDateTime.now, "f-code", None, Seq.empty, "payload")
   private val connector = new CustomsDeclareExportsConnector(config, httpClient)
 
+  implicit val defaultPatience: PatienceConfig =
```
Comment (PatrykRudnicki) [inline test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: First test take more time than default timeout

File: test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala [hunk 3]
```diff
@@ -62,9 +66,9 @@ class CustomsDeclareExportsConnectorIntegrationSpec
           )
       )
 
-      val response = await(connector.createDeclaration(newDeclaration))
```
Comment (PatrykRudnicki) [inline test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: I can't find await method in scalatest, this one was from hmrc test

File: test/services/model/AutoCompleteItemSpec.scala [hunk 4]
```diff
@@ -18,64 +18,64 @@ package services.model
...
     "map from document type" in {
-      AutoCompleteItem.fromDocumentType(List(DocumentType("description", "code"))) shouldBe List(
+      AutoCompleteItem.fromDocumentType(List(DocumentType("description", "code"))) mustBe List(
         AutoCompleteItem("description - code", "code")
       )
     }
 
     "map from national additional code" in {
-      AutoCompleteItem.fromNationalAdditionalCode(List(NationalAdditionalCode("code"))) shouldBe List(
+      AutoCompleteItem.fromNationalAdditionalCode(List(NationalAdditionalCode("code"))) mustBe List(
```
Comment (PatrykRudnicki) [inline test/services/model/AutoCompleteItemSpec.scala]: Our base spec extends MustMatchers, so probably better to stay with one approach

File: test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala (hunk 3 above)
Comment (ed0906) [reply test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: `import play.api.test.Helpers._`

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: ah, okay, I'll change it here :)

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: Changed, waiting for build and merge

File: test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala [hunk 5]
```diff
@@ -64,7 +69,7 @@ class CustomsDeclareExportsConnectorIntegrationSpec
 
       val response = await(connector.createDeclaration(newDeclaration))
 
-      response shouldBe existingDeclaration
+      response mustBe existingDeclaration
```
Comment (PauloPortugal) [inline test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: why `mustBe`?

Comment (PatrykRudnicki) [reply test/connectors/CustomsDeclareExportsConnectorIntegrationSpec.scala]: `UnitSpec` form `hmrctest` extends `Matchers`, our `UnitSpec` extends `MustMatchers` so we don't have access to `shouldBe`.

To have it we need to extends `Matchers` but IMO it doesn't make any sense to add additional `Matchers`. `mustBe` is consistent with actual approach in other tests.

**Pull request merged**